backend/controller/observability/fsm.go

```diff
@@ -0,0 +1,38 @@
+package observability
+
+import (
+	"context"
+	"fmt"
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/observability/metrics"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const fsmMeterName = "ftl.fsm"
+
+var fsmMeter = otel.Meter("ftl.fsm")
+
+var fsmCounters = struct {
+	instancesActive metric.Int64UpDownCounter
+}{}
+
+// TODO error logging and handling
+func InitFSMMetrics() {
+	fsmCounters.instancesActive, _ = fsmMeter.Int64UpDownCounter(
```

Comment:
`func InitFSMMetrics()` should return an error so it can be checked.

Reply:
agreed; just going for a quicker turn around on other aspects of the PR by leaving that as a TODO comment, will return to it before exiting draft mode on this PR

Reply:
added the error handling